test/views/vatDetails/VatDetailsViewSpec.scala

```diff
@@ -136,16 +137,18 @@ class VatDetailsViewSpec extends ViewBaseSpec with BeforeAndAfterEach {
       elementText(Selectors.vatRegNo) shouldBe s"VAT registration number (VRN): ${user.vrn}"
     }
 
-    "have the account details section" should {
+    "have the vat certificate section" should {
```

Comment:
I think this ViewSpec also needs tidying up in general. There seems to be some inconsistencies and repeated tests. For example, I wouldn't have thought we would need to repeat the all of the tests for the scenario where we hide the payment history section.